scripts/PushNuGet.psm1

```diff
@@ -78,14 +85,21 @@ function Push-NuGetPackage {
                 $ErrorActionPreference = 'Stop'
                 Set-StrictMode -Version Latest
 
+                if ($env:DOTNET_HOME -ne $null) {
```

Comment:
I don't think you need to repeat this if you use the `$using` variable scope. 

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_remote_variables